test/integration/scheduler_perf/config/pod-with-topology-spreading.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: spreading-pod-
+spec:
+  topologySpreadConstraints:
+    - maxSkew: 5
```

Comment:
Just a Q, not blocking the review: Why 5? Why not 4 or 6?

Reply:
It could be anything reasonably small that we expect users to actually use.